doc/ext/imgconverter.rst

```diff
@@ -0,0 +1,30 @@
+.. highlight:: rest
+
+:mod:`sphinx.ext.imgconverter` -- Convert images to appropriate format for builders
```

Comment:
For now, this extension seems it depends to imagemagick command line interface, so this extension seems to be for imagemagick.